public/API/API_GetTopTenUsers.php

```diff
@@ -8,6 +8,7 @@
  *    string     1                     name of the user
  *    int        2                     total points earned by the user
  *    int        3                     total RetroPoints ("white points") earned by the user
+ *    string     4                     ulid of the user
```

Comment:
Should this expound on what "ulid" is? The other files all say something along the lines of "stable identifier"

Reply:
Updated in latest, but the best I could come up with was:
```php
 *    string     4                     ulid (queryable stable unique identifier) of the user
```
Happy to update further if this is insufficient.